### PR TITLE
add config management

### PR DESCRIPTION
**Pull request by Alex-Izquierdo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,3 +54,27 @@ There is a simple script for generation:
...
+Configuration is defined in `ansible_events_qa/conf/settings.yaml`, is managed by [dynaconf library](https://dynaconf.readthedocs.io/).
+Supports a `.env` file in the root of the project which can have defined environment variables to override default settings.
+You can override it with `AEQE_{keyname}={keyvalue}`
+The configuration supports multiple environments, you can define it with the environment variable `AEQE_ENV_{environment}`
+
+### encrypted values
+
+The values can have encrypted values that are automatically decrypted. You must to define the environment variable `AEQE_FERNET_PASSWORD` if you have some encrypted value. The encrypted values must follow the pattern `key: "!fernat:[encrypted data]"`
+
+The encryption uses `cryptography.fernet` which is an easy and reliable simmetric encryption based on AES cipher.
```
Comment (ttuffin) [inline README.md:69]: simmetric > symmetric

File: README.md [hunk 2]
```diff
@@ -48,3 +54,27 @@ There is a simple script for generation:
...
+## Configuration
+
+Configuration is defined in `ansible_events_qa/conf/settings.yaml`, is managed by [dynaconf library](https://dynaconf.readthedocs.io/).
+Supports a `.env` file in the root of the project which can have defined environment variables to override default settings.
+You can override it with `AEQE_{keyname}={keyvalue}`
+The configuration supports multiple environments, you can define it with the environment variable `AEQE_ENV_{environment}`
+
+### encrypted values
+
+The values can have encrypted values that are automatically decrypted. You must to define the environment variable `AEQE_FERNET_PASSWORD` if you have some encrypted value. The encrypted values must follow the pattern `key: "!fernat:[encrypted data]"`
```
Comment (ttuffin) [inline README.md:67]: You must to define > You must define

Comment (Alex-Izquierdo) [reply README.md:67]: thanks

File: README.md [hunk 3]
```diff
@@ -48,3 +54,27 @@ There is a simple script for generation:
...
+## Configuration
+
+Configuration is defined in `ansible_events_qa/conf/settings.yaml`, is managed by [dynaconf library](https://dynaconf.readthedocs.io/).
+Supports a `.env` file in the root of the project which can have defined environment variables to override default settings.
+You can override it with `AEQE_{keyname}={keyvalue}`
+The configuration supports multiple environments, you can define it with the environment variable `AEQE_ENV_{environment}`
+
+### encrypted values
+
+The values can have encrypted values that are automatically decrypted. You must define the environment variable `AEQE_FERNET_PASSWORD` if you have some encrypted value. The encrypted values must follow the pattern `key: "!fernat:[encrypted data]"`
```
Comment (ttuffin) [inline README.md:67]:
```suggestion
You can create encrypted values that will be automatically decrypted. Before you can encrypt a value you must first specify the fernet password by defining the environment variable `AEQE_FERNET_PASSWORD`. If you do not have the fernet password, please contact a team member to acquire it.

You can then use the encrypted value by specifying it with the following pattern: `key: "!fernat:[encrypted data]"`
```

File: README.md [hunk 4]
```diff
@@ -48,3 +54,27 @@ There is a simple script for generation:
...
+You can use `aeqe` command to encrypt or decrypt values:
+
+```shell
+aeqe --encrypt "password"
+# gAAAAABjEPXUFt5Oc-ivGsv6SZ9goAp8X_WIIBRJvzvIm7n8AGn18BL9tsAEGW6P2BUN2GjOXyBG711dEGg2s-qOJVAvjR0m4g==
+aeqe --decrypt "gAAAAABjEPXUFt5Oc-ivGsv6SZ9goAp8X_WIIBRJvzvIm7n8AGn18BL9tsAEGW6P2BUN2GjOXyBG711dEGg2s-qOJVAvjR0m4g=="
+# password
+```
+
+If you don't have the fernet password, contact with some team member to adquire it.
```
Comment (ttuffin) [inline README.md:80]:
```suggestion
```
Relocated to the "encrypted values" section.

File: README.md [hunk 5]
```diff
@@ -48,3 +54,27 @@ There is a simple script for generation:
 ```
 ./utils/api-client-generator/generate.sh
 ```
+
+## Configuration
+
+Configuration is defined in `ansible_events_qa/conf/settings.yaml`, is managed by [dynaconf library](https://dynaconf.readthedocs.io/).
```
Comment (ttuffin) [inline README.md:60]:
```suggestion
Configuration is defined in `ansible_events_qa/conf/settings.yaml` and is managed by [dynaconf library](https://dynaconf.readthedocs.io/).
```

**Review by ttuffin — CHANGES_REQUESTED** [review]
Minor changes to readme

**Review by ttuffin — APPROVED** [review]

**Comment by Alex-Izquierdo** [comment]
Rebased and fixed format. Review again pls. @ttuffin

**Review by ttuffin — APPROVED** [review]

**Pull request merged**